main.py

```diff
@@ -22,9 +20,12 @@ def new_command():
 
 
 @app.command()
-def if_statements():
+def if_statements(directory_path):
     """Determine number of if statements in a Python directory."""
-    print("In progress")
+    cast_dict = generate_trees(directory_path)
+    if_statements_directory = counter.match_ifstatements(cast_dict)
+
+    print(if_statements_directory)
```

Comment:
Once you create a function to determine totals, please integrate into the CLI. When this command is run, it should output the total number of if statements in a directory.